config/application.rb

```diff
@@ -1,29 +1,36 @@
 require_relative "boot"
 
-require "active_job/railtie"
+require "rails"
+# Pick the frameworks you want:
 require "active_model/railtie"
+require "active_job/railtie"
 require "active_record/railtie"
+# require "active_storage/engine"
 require "action_controller/railtie"
+# require "action_mailer/railtie"
+# require "action_mailbox/engine"
+# require "action_text/engine"
 require "action_view/railtie"
+# require "action_cable/engine"
 require "sprockets/railtie"
-
-require "tilt/erubi"
+require "rails/test_unit/railtie"
 
 # Require the gems listed in Gemfile, including any gems
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
 module CollectionsPublisher
   class Application < Rails::Application
```

Comment:
```suggestion
  class Application < Rails::Application
    config.load_defaults 6.0
```
We should add this line to use all the new rails 6 defaults. If not all can be used, we should still add this line and then override below the ones we don't want.

Reply:
:+1: